assets/scss/templates/_guidelines.scss

```diff
@@ -100,12 +100,18 @@ body {
 .section-module {
   width: 100%;
   max-width: $dp-wrapper-max-width;
+  border-bottom: 1px solid $dp-color-silver;
   margin: 0 auto;
   padding: 50px 0 100px;
```

Comment:
In this line maybe you can use class for spaces

Reply:
Later I will refactor the entire layout of the library, to improve the menu and all the blocks. This will be another pull request.

Reply:
Ok!

Reply:
Disculpen, no entendí bien por que en este caso se decidió no usar los espacios predefinidos. Me podrán explicar?

Reply:
@andresmoschini no corresponde a esta tarea. Las variables de espacio y sizes fueron creadas mucho después del armado del layout de la librería. Mas adelante cuando saquemos las prioridades voy a hacer un refactor del layout de la librería para mejorar la legibilidad y la navegación entre componentes. El menú nos quedo chico, y así poder implementar todas las variables que tenemos disponibles.